src/components/CryptoCoinItem.tsx

```diff
@@ -0,0 +1,98 @@
+import React, {useEffect, useState} from 'react';
+import {ActivityIndicator, ScrollView, Text} from 'react-native';
+import styled from 'styled-components/native';
+import Avatar from './Avatar';
+import {CryptoApiResponse, CryptoCoin} from './types';
+
+export const CryptoCoinItem = () => {
+  const [isLoading, setLoading] = useState<boolean>(true);
+  const [crypto, setCrypto] = useState<CryptoCoin[]>([]);
+
+  const getCrypto = async () => {
+    try {
+      const response = await fetch('https://data.messari.io/api/v1/assets');
```

Comment:
the conecction with the api is outside the scope, it should be implemented in the correct PR